benchmarks/microbenchmarks/utils.py

```diff
@@ -291,6 +292,15 @@ def string_to_config(
         else:
             granularity = PerTensor()
         return Float8DynamicActivationFloat8WeightConfig(granularity=granularity)
+    if "gemlitewo" in quantization:
```

Comment:
gemlite can be 4 bit or 8 bit, should probably specify that this is for 4 bit

https://github.com/pytorch/ao/blob/main/torchao/quantization/quant_api.py#L968